optaplanner-examples/src/main/java/org/optaplanner/examples/taskassigning_listvariable/domain/TaskAssigningSolution.java

```diff
@@ -40,13 +40,12 @@
     private List<TaskType> taskTypeList;
     @ProblemFactCollectionProperty
     private List<Customer> customerList;
-    @ValueRangeProvider(id = "employeeRange")
+    @ValueRangeProvider(id = "taskRange")
```

Comment:
General question: is `taskassigning_listvariable` the pattern we want to be applying here?
I'd rather each example used either one or the other - and preferrably only one lone example would be picked to stay on the old chained variable. Perhaps something trivial, as TSP.

Reply:
The answer to the question is "no". When the feature is done, before it goes to `main`, everything in `taskassigning_listvariable` will be moved back to `taskassigning`. Effectively the existing task assigning example will be upgraded to list variable and there will be no other task assigning variant.

The rest sounds like a good idea. I'll be happy to see most of the existing examples that use chained variable upgrade to the list variable when they're ready.